replicaset_test.go

```diff
@@ -588,15 +600,15 @@ type MongoIPV6Suite struct {
 var _ = gc.Suite(&MongoIPV6Suite{})
 
 func (s *MongoIPV6Suite) TestAddRemoveSetIPv6(c *gc.C) {
-	c.Skip("Skipping test until mgo issue 22 is fixed")
-
 	root := newServer(c)
 	defer root.Destroy()
 	dialAndTestInitiate(c, root, ipv6GetAddr(root))
+
 	assertAddRemoveSet(c, root, ipv6GetAddr)
 }
 
 func (s *MongoIPV6Suite) TestAddressFixing(c *gc.C) {
+	c.Skip("Skipping test until rewritten for good and bad IPv6 addr formats")
```

Comment:
In fact, I'm not sure we now need to skip this, with the fix in the Initiate function. Thanks for the catch.